Bottom: 27b6d5f9673e577e84d3910dabb0cfca9ab14b17
Top:    adef8b7b8dea8be10778d778c9838c153dc26580
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 20:59:50 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 8397978..a51d167 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1838,8 +1838,8 @@ ScrollAction (int argc, char **argv, int x, int y)
   else
     AFAIL (scroll);
 
-  ghid_port_ranges_pan (dx, dy, TRUE);
   notify_crosshair_change (false);
+  ghid_port_ranges_pan (dx, dy, TRUE);
   MoveCrosshairRelative (dx, dy);
   AdjustAttachedObjects ();
   notify_crosshair_change (true);
